system-integration/systemd/service.tendrl_server.elementary_check.rst

```diff
@@ -7,8 +7,8 @@ Elementary checks of tendrl-apid service
 Description
 ===========
 
-Elementary check of tendrl-apid systemd service unit file. This test case
-doesn't change any state in any way.
+Elementary check of tendrl-apid, tendrl-node-agent and tendrl-monitoring-integration 
```

Comment:
The name of the component and also the name of the service is ``tendrl-api`` not ``tendrl-apid``.